include/seqan/sequence/string_set_concat_direct.h

```diff
@@ -429,7 +487,7 @@ template <typename TString, typename TSpec, typename TSize, typename TExpand >
 inline typename Size<StringSet<TString, Owner<ConcatDirect<TSpec> > > >::Type
 resize(StringSet<TString, Owner<ConcatDirect<TSpec> > > & me, TSize new_size, Tag<TExpand> tag)
 {
-    if (new_size < length(me.limits))
+    if (static_cast<typename Size<StringSet<TString, Owner<ConcatDirect<TSpec> > > >::Type>(new_size) < length(me.limits))
```

Comment:
I know this is kind of pedantic, but it should be the Size type of StringSetLimits of StringSet ...
I know it is basically the same, but who knows what will be changed in the future.
:trollface: